spec/unit/lib/temporal/connection/converter/codec/base_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'temporal/connection/converter/codec/chain'
+
+describe Temporal::Connection::Converter::Codec::Base do
```

Comment:
Testing the base class seems strange to me since no one is expected to use this directly. It will always be extended, right? This might be a further indication that these non-abstract methods should simply be moved into `payloads.rb` instead.

Reply:
hmm, i don't disagree, just was following the pattern that was already in place for `from_payloads` and `to_payloads`. What do you think? Im easy either way.

Reply:
if we do remove, ill remove the unit tests